src/highlight.js

```diff
@@ -0,0 +1,18 @@
+const cards = document.querySelectorAll(".number-card")
+let randomNumbersInSequence = randomNumbersHolder.sort((function(a, b){return a-b}))
+let correctNumbersHolder = []
+
+cards.forEach(function (card) {
+  card.addEventListener("click", (e) => {
+    if (e.target.textContent == Math.min(...randomNumbersInSequence)) {
```

Comment:
instead of using Math.min, you just need to access the first number in the array, as you have already put them in order, this could just be `randomNumbersInSequence[0]`